Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -1047,6 +1047,22 @@ function vtkOpenGLImageMapper(publicAPI, model) {
         vtkErrorMacro('Reformat slicing not yet supported.');
       }
 
+      // For now, fix the image mapper, but we should probably fix the
+      // vtkImageArrayMapper as well
+      if (!model.renderable.isA('vtkImageArrayMapper')) {
+        const { offset, scale } = model.openGLTexture.computeScaleOffsets(
+          numComp,
+          scalars
+        );
+
+        model.openGLTexture.setUseHalfFloat(
+          dataType,
+          offset,
+          scale,
```

Comment:
shouldn't you simply have to pass `model.renderable.getPreferSizeOverAccuracy()` to `create2DFromRaw` below ?

Reply:
Or maybe add a `createFilterable2DFromDataArray` or something like that...